common/main/en_CA.xml

```diff
@@ -1854,10 +1854,11 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 						<dateFormatItem id="Ehms">↑↑↑</dateFormatItem>
 						<dateFormatItem id="EHms">↑↑↑</dateFormatItem>
 						<dateFormatItem id="Gy">↑↑↑</dateFormatItem>
-						<dateFormatItem id="GyMd">↑↑↑</dateFormatItem>
-						<dateFormatItem id="GyMMM">↑↑↑</dateFormatItem>
-						<dateFormatItem id="GyMMMd">↑↑↑</dateFormatItem>
-						<dateFormatItem id="GyMMMEd">↑↑↑</dateFormatItem>
+						<dateFormatItem id="GyMd">y-MM-dd G</dateFormatItem>
+						<dateFormatItem id="GyM">y-MM G</dateFormatItem>
+						<dateFormatItem id="GyMMM">y MMM G</dateFormatItem>
```

Comment:
What are we changing the medium-style formats here? I thought those were supposed to inherit from en. It is only the short (numeric) dates that should have y MM dd order.

Reply:
Good catch, we only want to change the MM or M patterns.

My list (https://github.com/unicode-org/cldr/pull/2759#issuecomment-1452934163) just had the numeric formats. Not sure where the MMM's snuck in.

Reply:
FYI: there are other languages that have different order for numeric dates than they do for non numeric (eg MMM+), like Swedish.

Reply:
OK, will drop the MMM.  I thought they also needed to be in the same order.